app/routes/document-configure-response.js

```diff
@@ -22,7 +22,7 @@ module.exports = [{
 
       let prompts = []
       if (selectedModel && selectedModel !== '') {
-        prompts = await getPrompts(request, 'mcu', selectedModel, 'correspondence')  // 'briefing'
+        prompts = await getPrompts(request, 'mcu', selectedModel, 'correspondence') // 'briefing'
```

Comment:
Could we take out the comment at the end of this line?

Reply:
comment removed